### ISSUE TITLE
feat: 💯 Accessibilty

### ISSUE THREAD

**Issue by thekauer** [thekauer/struktorize#47]
- Screen reader shall read the sctuctogram is if it was just code
- First tab: jump to content
- Alt text for everything
- keyboard navigation should use tab to chose between main and menu, and arrow inside main and menu
- screen reader should be able to read structograms as human speach
